pkgs/tools/security/hashrat/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hashrat";
+  version = "1.15";
+
+  src = fetchFromGitHub {
+    owner = "ColumPaget";
+    repo = "Hashrat";
+    rev = "v${version}";
+    hash = "sha256-+3IPCJS++7CE0ZrJb62LCRrAn2J4uCF3a1oOzDoOW0w=";
+  };
+
+  configureFlags = [ "--enable-xattr" ];
```

Comment:
Does removing this for darwin make it build? Otherwise just mark it broken or change platform if upstream has no darwin support.

Reply:
Fix right now.